Vagrantfile

```diff
@@ -1,36 +1,29 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
-VAGRANTFILE_API_VERSION = '2'.freeze
-
 Vagrant.require_version '>= 1.5.0'
 
-Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  # All Vagrant configuration is done here. The most common configuration
-  # options are documented and commented below. For a complete reference,
-  # please see the online documentation at vagrantup.com.
-
-  config.vm.hostname = 'stackstorm-berkshelf'
+# Autoinstall for vagrant-omnibus plugin.
+# See https://github.com/chef/vagrant-omnibus
+unless Vagrant.has_plugin?('vagrant-omnibus')
+  system('vagrant plugin install vagrant-omnibus') || exit!
```

Comment:
I found we have another abandoned Vagrantfile PR.
So I took some good things from that place https://github.com/StackStorm/chef-stackstorm/pull/21/files#diff-23b6f443c01ea2efcb4f36eedfea9089R4, thanks to @jvrplmlmn